packages/ui/src/label.tsx

```diff
@@ -0,0 +1,51 @@
+import { forwardRef } from 'react'
+import * as RadixLabel from '@radix-ui/react-label'
+import type { ComponentPropsWithoutRef } from 'react'
+import { tv } from 'tailwind-variants'
+import type { VariantProps } from 'tailwind-variants'
```

Comment:
This could be `import {tv, type VariantProps } from 'tailwind-variants'`

You can use this VSCode setting:
```json
"typescript.preferences.preferTypeOnlyAutoImports": true,
```